src/Printer_CartridgeInfo.php

```diff
@@ -63,14 +63,52 @@ public function getInfoForPrinter(Printer $printer)
         return $info;
     }
 
+    public function getCartridgeTagInfoForPrinter(Printer $printer)
```

Comment:
Could the parameter be changed to just be the printer ID? This way, if the method is used for something else we don't need to create a printer object if we already have the ID.

Reply:
I can do this, but as the calling function receives already a printer object as parameter, I wonder if it is useful... Do you want me to change it however?

Reply:
Useless you mean?